lib/WebDriverWindow.php

```diff
@@ -72,6 +77,20 @@ public function getSize()
         );
     }
 
+    /**
+     * Minimizes the current window if it is not already maximized
+     *
+     * @return WebDriverWindow The instance.
+     */
+    public function minimize()
+    {
+        if ($this->isW3cCompliant) {
+            $this->executor->execute(DriverCommand::MINIMIZE_WINDOW, []);
+        }
```

Comment:
As minimize is W3C specific, shouldn't we throw an exception if it is called in non-w3c mode?

```php
if (!$this->isW3cCompliant) {
    throw new UnsupportedOperationException('Minimize window is only supported in W3C mode');`
}

$this->executor->execute(DriverCommand::MINIMIZE_WINDOW, []);
```